Student/StudentUITests/Assignments/AssignmentDetailsTests.swift

```diff
@@ -34,17 +34,11 @@ class AssignmentDetailsTests: StudentUITestCase {
         return assignment
     }
 
-    override func show(_ route: String) {
-        super.show(route)
-        sleep(1)
-    }
```

Comment:
If it still works with this removed, that is awesome! It was always sad this needed an extra sleep.